index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Stuart Dambi | Fullstack Developer</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <script src="https://kit.fontawesome.com/1a66ebfe2a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header class="header">
+    <ul>
+      <li><a href="#" class="developer">Dambi Stuart</a></li>
+      <li><a href="#"><i class="fa-solid fa-bars 2x"></i></a></li>
```

Comment:
The menu button is part of the navigation, so make sure to put this in a `nav` semantic tag.

Reply:
@codedbystuart Kindly check this comment 🤗